src/deltacode/utils.py

```diff
@@ -264,6 +264,24 @@ def check_moved(added_sha1, added_deltas, removed_sha1, removed_deltas):
     if added_deltas[0].new_file.name == removed_deltas[0].old_file.name:
         return True
 
+def get_notice_for_cli():
+    """
+    Retrieve the notice text from the NOTICE file for display in the Command Line .
+    """
+    notice_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'NOTICE')
+    notice_text = open(notice_path).read()
+
+    delimiter = '\n\n\n'
+    [notice_text, extra_notice_text] = notice_text.split(delimiter, 1)
+    extra_notice_text = delimiter + extra_notice_text
+    
+    delimiter = '\n\n '
+    [notice_text, acknowledgment_text] = notice_text.split(delimiter, 1)
+    acknowledgment_text = delimiter + acknowledgment_text
+
+    notice = acknowledgment_text.strip().replace('\n', '')
+
+    return notice
```

Comment:
Why do we need this?

Reply:
@majurg  the screen shot which I provided it here is the output of this part (the green area)

> @majurg kindly review this, I made few changes
> 
> ![Screenshot from 2020-03-23 02-01-23](https://user-images.githubusercontent.com/44219625/77260243-d7dd4a00-6cac-11ea-97d2-27e327fed99a.png)
> 
> Now I provided the the deltas which are modified in the CLI (deltas score,factors,old file ,new file...),so this is actually providing a bit more details to the end user about the deltas

Reply:
> @majurg sir I updated it now
> ![Screenshot from 2020-03-13 22-29-41](https://user-images.githubusercontent.com/44219625/76643397-f2ac1200-657a-11ea-9d93-d628275d8dcb.png)

@majurg now this is the output when I do not use this additonal function

Reply:
> Why do we need this?

It is added to make the notice output in the CLI look better

Reply:
I do not want the notice displayed in the section.